draft-ietf-quic-multipath.md

```diff
@@ -1343,7 +1343,7 @@ Maximum Path Identifier:
 QUIC transport error codes are 62-bit unsigned integers
 following {{QUIC-TRANSPORT}}.
 
-The following additional QUIC error codes are defined for use in the PATH_ABANDON frame.
+In addition to NO_ERROR(0x0) following {{QUIC-TRANSPORT}}, the following QUIC error codes are defined for use in the PATH_ABANDON frame.
 
```

Comment:
Should we add provisional values for the two error codes in the IANA section, to enable interop testing?

Reply:
I support @huitema 's idea, we need two experimental error codes value here for interop testing.